Dockerfile

```diff
@@ -13,19 +13,28 @@ RUN dnf install -y gcc git make xz-devel
 WORKDIR /tmp
 
 RUN git clone https://github.com/ipxe/ipxe.git && \
-      cd ipxe && \
-      git reset --hard $IPXE_COMMIT_HASH && \
-      cd src && \
-      ARCH=$(uname -m | sed 's/aarch/arm/') && \
-      # NOTE(elfosardo): warning should not be treated as errors by default
-      NO_WERROR=1 make bin/undionly.kpxe "bin-$ARCH-efi/snponly.efi"
+     cd ipxe && \
+     git reset --hard $IPXE_COMMIT_HASH && \
+     cd src && \
+     ARCH=$(uname -m | sed 's/aarch/arm/') && \
+     # NOTE(elfosardo): warning should not be treated as errors by default
+     NO_WERROR=1 make bin/undionly.kpxe "bin-$ARCH-efi/snponly.efi"
 
 COPY prepare-efi.sh /bin/
 RUN prepare-efi.sh centos
 
+# build actual image
 FROM $BASE_IMAGE
 
-ENV PKGS_LIST=main-packages-list.txt
+LABEL org.opencontainers.image.description="Container image to run OpenStack Ironic as part of Metal³"
+LABEL org.opencontainers.image.documentation="https://book.metal3.io/ironic/introduction"
```

Comment:
Any better documentation urls? I'm OK with this or https://book.metal3.io/ironic/ironic-container-images as well.

Reply:
https://book.metal3.io/ironic/introduction is good for me